src/librustc_lint/builtin.rs

```diff
@@ -621,12 +621,6 @@ impl EarlyLintPass for AnonymousParameters {
     }
 }
 
-declare_lint! {
-    DEPRECATED_ATTR,
```

Comment:
Just removing these lints will cause users who mention them in allow/warn/deny statements to produce warnings about unknown lints. Does rustc have a removed lint list like clippy does?

Reply:
Yes, there's a list of removed lints, I'll add these these into it.